git-ftp

```diff
@@ -811,6 +816,45 @@ handle_remote_protocol_options() {
 	[ "$CURL_INSECURE" != "0" ] && REMOTE_CMD_OPTIONS="$REMOTE_CMD_OPTIONS -k"
 }
 
+extract_folder_clone() {
+    local fulladdress=$1
+    #check if the last char is /, in this case eliminate it!
+    length=${#filename}
+    ((length--))
+    [ "${fulladdress:$length:1}" == "/" ] && fulladdress="${fulladdress%?}"
+
+    FOLDER_CLONE=${fulladdress##*/}
+}
+
+set_up_clone_folder() {
+    
+    [ -z $URL ] && echo Error: give a url to clone. && exit
+    [ $FOLDER_CLONE == "." ] && extract_folder_clone $URL
+    
+    if [ -d "$FOLDER_CLONE" ]; then
+        echo "Folder $FOLDER_CLONE already exists, Do you want to overwrite it? [yes/no]. All change will be lost"
+        local OVERWRITE=
+        read OVERWRITE
+        if [[ $OVERWRITE == "y"* ]]; then rm -rf $FOLDER_CLONE
+        else exit 
+        fi
+    fi
+    
+    mkdir $FOLDER_CLONE && \
+    cd $FOLDER_CLONE && \
+    git init
+
+    git config git-ftp.user $REMOTE_USER
```

Comment:
You can't do that if the commands above failed. You also don't know if people want the credentials stored in the config or keep them secret.

Storing options in the config has been a feature request before and it would be nice as a separate feature. But I wouldn't include it in this pull request.